js/map.js

```diff
@@ -320,11 +320,16 @@ function customMarkerGenerateJSON() {
     '     "dateAdded": "' + currentJSONDate + '"\n' +
     '   }\n</pre>';
 
-  document.getElementById('customMarkerJSONDiv').innerHTML = sJSON + '<input type="button" value="Start Over" onclick="clearWindows();" />';
+  document.getElementById('customMarkerJSONDiv').innerHTML = sJSON + '<input type="button" class="button-primary" value="Submit Your Node" onclick="submitJson();"> <input style="margin-left:50px;" type="button" value="Start Over" onclick="clearWindows();" />';
 
 
 }
-
+function submitJson() {
+  var msg = 'I would like to add my node to the Toronto Mesh node list. I\'ve completed the JSON object below describing my node.\n\n```\n' + document.getElementById('jsonData').innerHTML + '```';
+  var name = document.getElementById('customMarkerName').value;
+  name=encodeURI(name);
+  document.location='https://github.com/tomeshnet/node-list/issues/new?labels=New+Node&title=New+node+report+' + name + '&body=' + encodeURI(msg);
```

Comment:
Just confirming -- do we want the title as "report"? Could it be "map submission"?
Also RE: label -- totally great call! but again wondering if the important aspect isn't that this is a map/website submission rather than a new one?

Reply:
I was going generic, since this is an entry into the node list not the map. The map just happens to use this data to plot. 

But since the the map is the central focus, the information will be accessible in the most public manner, and that we are taking them from the map to this post, I'm ok with denoting that its a map addition.